sample/F8/event

```diff
@@ -0,0 +1,51 @@
+{  
+   "id":180,
+   "name":"F8-Facebook Developer Conference 2017",
+   "latitude":37.329008,
+   "longitude":-121.888794,
+   "location_name":"San Jose Convention Center, 150 W San Carlos St  San Jose  CA USA 95113 ",
+   "start_time":"2017-04-18T10:00:00",
+   "end_time":"2017-04-19T17:00:00",
+   "timezone”:”US/Pacific”,
```

Comment:
Quote issue again @mahikaw 
Please be careful with quotes, this happens way too much